pwndbg/commands/stack.py

```diff
@@ -26,7 +26,7 @@ def retaddr() -> None:
     # Find all of them on the stack
     start = stack.vaddr
     stop = start + stack.memsz
-    while addresses and start < sp < stop:
+    while addresses and start <= sp < stop:
```

Comment:
I guess this is to cover the case when the return address is pushed to the stack but we haven't pushed a frame pointer yet, so `$rsp` points to the return address and without the `<=` change we wouldn't print it (cc: @disconnect3d )? If so, it would be good to add a test for this.

Reply:
pinging @disconnect3d about this one again because i'm not totally sure